packages/cfd/src/Helpers/constants.ts

```diff
@@ -15,10 +15,14 @@ const getTopUpConfig = () => {
     };
 };
 
-const getPlatformDXTradeDownloadLink = (platform: 'ios' | 'android') => {
+const getPlatformDXTradeDownloadLink = (platform: 'ios' | 'android' | 'huawei') => {
     if (platform === 'ios') {
         return DXTRADE_IOS_APP_URL;
     }
+    if (platform === 'huawei') {
```

Comment:
@hamid-deriv Just a suggestion, Maybe using `switch` would be a better fit for here.